ui/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "b-api",
+  "version": "1.0.0",
+  "description": "a test",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/karma/bin/karma start --single-run",
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "bootstrap": "^3.4.1",
+    "bootstrap-datepicker": "^1.9.0",
+    "css-loader": "^3.5.3",
+    "d3": "^4.2.6",
+    "d3-geo-projection": "^1.0.3",
```

Comment:
Is d3 actually used for anything? I remember I made a demo at one point, but I don't think its there any more. Remove?

Reply:
no, it is not used. will remove stuff that is not used, when I get everything to work. I have one test that fails because we do not find level2 data using the browser...and I do not see the problem yet.